### PR TITLE
Homework #1

### PR DESCRIPTION
**Pull request by Konstantin343**

### CODE REVIEW

File: client/cypress/integration/HomePage.spec.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+describe('My First Test', () => {
+    it('Does not do much!', () => {
```
Comment (volekerb) [inline client/cypress/integration/HomePage.spec.js]: Better to go with proper description here, smth like:
'User should be able to sign up'

File: client/src/tests/e2e/PlacesList.spec.mjs [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import {test, expect} from '@playwright/test';
+import {addNewList, addNewPlace, submitCredentials} from "./Steps.mjs";
+import {config} from 'dotenv'
+config({ path: './.env.test' });
+
+let suffix
+let login
+let password
+
+const host = 'http://localhost'
```
Comment (volekerb) [inline client/src/tests/e2e/PlacesList.spec.mjs]: hm I thought you defined them in env variable

File: client/src/tests/e2e/PlacesList.spec.mjs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+const port = 3000
+
+test.describe('Places lists', () => {
+    test.beforeEach(async ({page}) => {
+        if (!login && !password) {
+            login = 'login' + Math.random()
+            password = 'password' + Math.random()
+            await page.goto(`${process.env.E2E_TESTS_URL}/sign-up`)
```
Comment (volekerb) [inline client/src/tests/e2e/PlacesList.spec.mjs]: btw in playwright you can define playwright config with the base URL in the case you don't have to repeatably write  process.env.*

you would have just page.goto('/sign-up')

playwright.config.ts could be like:

const config: PlaywrightTestConfig = {
  use: {
    baseURL: getBaseUrl(),
    headless: true,
    viewport: { width: 1280, height: 720 },
    ignoreHTTPSErrors: true,
  },
}

where getBaseUrl could be like util function:

export const getBaseUrl = () => {
  return process.env.BASE_URL ?? default_base_url
}

**Review by volekerb — APPROVED** [review]

File: client/cypress/integration/HomePage.spec.js (hunk 1 above)
Comment (Konstantin343) [reply client/cypress/integration/HomePage.spec.js]: Oh, just forgot to change the name from the cypress example)

**Pull request merged**